### ISSUE TITLE
Add an Example of Simple ANN model on MNIST Dataset Using Pytorch (Deep Learning)

### ISSUE THREAD

**Issue by ManthanShettigar** [DSC-ChitkaraUniv/Hacktoberfest_2021#37]
Hello Maintainer

- [ ]  Load Dataset : The MNIST database of handwritten digits has a training set of 60,000 examples, and a test set of 10,000 examples.
- [ ]  Build Model : ANN(Artificial Neural Network).
- [ ] Use Activation Function : ReLu
- [ ] Train on GPU 
- [ ] Use Adam Optimizer
- [ ] Evaluate Model by plotting Confusion Matrix and Loss
- [ ] Show Wrong predictions/misclassified Samples

Please assign me to work on this issue
